### ISSUE TITLE
[New Rule] Replace `hasattr` if else with `getattr` when appropiate

### ISSUE THREAD

**Issue by Skylion007** [MartinThoma/flake8-simplify#180]
## Explanation

This code is more succinct, readable, and slightly more efficient. We recently opened a few PRs to add this to the PyTorch codebase. Exhibit A: https://github.com/pytorch/pytorch/pull/100360 If you are aware of a different flake8-plugin / ruleset that implements this rule, feel free to recommend it.

## Example

```python
# Bad
...  # your bad code goes here
X.Y if hasattr(X, "Y") else Z

# Good
...  # your improved code goes here
getattr(X, "Y", Z)
```
